client/src/components/QuickReplies/__tests__/__snapshots__/index.spec.js.snap

```diff
@@ -6,29 +6,14 @@ exports[`components/<QuickReplies /> should render component with data 1`] = `
 >
   <p />
   <QuickReplyItem
-    data={
```

Comment:
I need to add here values for props, otherwise, the snapshot is not really useful...